frontend/server/src/Controllers/Course.php

```diff
@@ -3399,6 +3398,7 @@ public static function getIntroDetails(\OmegaUp\Request $r): array {
                 $privacyStatement['statementType'] = $statementType;
                 if (!is_null($statement)) {
                     $privacyStatement['gitObjectId'] = $statement['git_object_id'];
+                    $statements['privacy'] = $privacyStatement;
```

Comment:
```suggestion
                    $statements['privacy'] = [
                        'markdown' => 'privacyStatementMarkdown',
                        'statementType' => $statementType,
                        'gitObjectId' => $statement['git_object_id'],
                    ];
```

igual abajo. así es más fácil de entender qué contiene.

Reply:
Cierto, es mucho más legible. Se cambian ambos